app/src/main/res/values/strings.xml

```diff
@@ -290,4 +290,7 @@
         <item quantity="one"><xliff:g>%1$d</xliff:g> card (<xliff:g id="archivedCount">%2$d</xliff:g> archived)</item>
         <item quantity="other"><xliff:g>%1$d</xliff:g> cards (<xliff:g id="archivedCount">%2$d</xliff:g> archived)</item>
     </plurals>
-</resources>
\ No newline at end of file
+    <string name="previousCard">Previous card</string>
+    <string name="nextCard">Next card</string>
+    <string name="cardNoAdditionalInfo">This card has no additional data. When you add a note, add it to a group, set balance or set expiry it will appear here.</string>
```

Comment:
@comradekingu Knowing you, you'll probably have an opinion on this. So I'm hereby pinging you so we can discuss if this string needs rephrasing before it becomes editable :)

For context, this appears when you hit the info button on a card with no user data set on it (no note, expiry, etc.)

![image](https://user-images.githubusercontent.com/1885159/179838167-dec04e9b-3199-41d8-a51b-6083164a1581.png)

Reply:
I thought about just a line-shift, and then a more direct approach which is suggested for now.
Ideally there would not be a difference between the editable mode and the viewable one.
This solves the "exit without saving" when no changes are made.
The three different "Card" "Alternatives" "Images" could also be one screen.

Really what I would want is for every group to have a scrollable screen with one fully shown card.
Card1 +  | Group 2
Card 2 +     …
Card 3 -
Clickable name
Clickable note (Clickable info from "alternatives" is available here)
Clickable barcode image that opens camera (has the setting for type too, and additional images menu)
Clickable card number ID (clickable "Same bar code value")
Card 4 +
Card 5 +

This would make the app instantly usable after opening, with zero additional clicks to use the "main card".

Reply:
I'm really not interested in refactoring the whole view/edit model. That is out of scope and a lot of work for no real visible benefit (and will just increase the risk of accidental edits). The current model works fine.